share/pot/signal.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/sh
+# shellcheck disable=SC3033,SC3040,SC3043
+:
+
+signal-help()
+{
+	cat <<-"EOH"
+	pot signal [-hv] [-s signal] [-P pid] [-m pattern]-p pot
+	  -h print this help
+	  -v verbose
+	  -f : force - always returns success, even if no process could be killed
+	  -s signal : the symbolic name of the signal to send to the process,
+	              defaults to SIGINFO. Use `killall -l` to get a list of
+	              supported signals.
+	  -P pid : the pid inside the pot to send the signal to.
+	           for non-persistent pots this defaults to
+	           the pid of the main process
+          -m pattern : A pattern to match (overrides -P)
```

Comment:
It seems that there is some misalignment for `-m pattern`.
Tabs vs Space, maybe?